help/en/html/hardware/tams/index.shtml

```diff
@@ -72,7 +72,8 @@ The USB port works under Linux and Windows - both have been tested.
 
 <a name="documentation"></a><h2>Documentation</h2>
 <h3>Third Party info</h3>
-<p>The <a href="http://tams-online.de/MasterControl">TAMs.de manufacturer's website</a>
+<p>Web site and info mainly in German, user manuals also in English (and other languages) for download</p>
+<p>The <a href="http://tams-online.de/MasterControl">tams.de manufacturer's website</a></p>
```

Comment:
"tams.de" is an unrelated site. You should probably change "tams.de" to "Tams" since the firm is "Tams Electronic GmbH" unless TAMS is a product from Tams (either way the capitalization needs to be consistent).